### ISSUE TITLE
this.subscribe seems not running again without changing arguments

### ISSUE THREAD

**Issue by dohomi** [Urigo/angular-meteor#1338]
Hi,

I just had an issue where my `this.subscribe` doesn't execute again if called with the same arguments:

``` javascript
this.autorun(() => {
            this.tracker.depend();
            if (subsHandle) {
                subsHandle.stop();
            }
            // this.subscribe is not working if the function is running again 
            subsHandle = this.subscribe('eventsByOwnerAndDate', () => [this.start, this.end], () => {
             (...)
             }
```

`this.start` and `this.end` are the same arguments but the subscription doesn't run again. I had to change the `this.subscribe` into `Meteor.subscribe`. 

For reproduction: Just set up a tracker which calls the `autorun` again but don't change the subscription arguments. Even the handle is stopped the subscription does not execute the callback.

This works as expected:

``` javascript
            subsHandle = Meteor.subscribe('eventsByOwnerAndDate', start, end, () => {
```

Cheers Dominic

**Comment by DAB0mB** [comment]
@dohomi Why would you use a subscribe inside an autorun scope? The reason why it happens is because `$scope.subscrube()` uses `Meteor.subscribe` inside an autorun, so once there is a subscription in a nested autorun, the outer computation will stop our subscription. Can't you just subscribe outside the autorun?

**Comment by dohomi** [comment]
Thanks for the hint about the `autorun`. But still: even if I use following code the subscription is not running again even the `this.tracker.depend()` is triggert:
```javascript
const eventHandle = this.subscribe('eventsByOwnerAndDate', () => {
            this.tracker.depend();
            var start = this.view && this.view.start;
            var end = this.view && this.view.end;
            return [start, end];
        }, () => {
         console.log("subscription finished") // never getting called once the subs is finished. start + end are same values but this.tracker.depend() should trigger a rerun
        });
```

**Comment by DAB0mB** [comment]
@dohomi I just don't get you, what exactly are you trying to achieve here?! Why the `this.tracker.depend()`? I think you misunderstood the concept of computations and how they work... Or maybe I'm just having a hard time understanding you.

**Comment by dohomi** [comment]
The tracker.depend() I’m using because I update the subscription through a method call. And there is more complexity involved because I’m using jQuery full calendar for rendering it. Important for me is: if I trigger the autorun or subscription through a reactive source I await that the subscription runs again - which unfortunately does not happen if I use this.autorun rather than Meteor.autorun.

Thanks for the heads up. Cheers

**Comment by Urigo** [comment]
closing due to inactivity and cleanup after tons of new updates in the recent month.
Feel free to open a new issue if you are still affected
